_posts/2021-04-27-Optimising-ux-for-software.markdown

```diff
@@ -7,70 +7,65 @@ author: Lucy Sharp
 image: /img/User_Journey_Map.jpeg
 ---
 
-My baptism into the cult of Mechanical Rock kicked off with a sword fight (legally, I can't say more) and it got me thinking about the relationship between UX and DevOps. Particularly, how DevOps drives the evolution of UX design.
+My baptism into the cult of Mechanical Rock kicked off with a sword fight (legally, I can't say more) and it got me thinking about the relationship between UX and DevOps.
 
-Sword fight aside, there is opportunity to accelerate technology delivery through the reciprocal relationship between UX and DevOps especially as UX provides a competitive advantage to software delivery.
+I wanted to better understand this relationship, considering DevOps is Mechanical Rock's way of working. So I started researching and listening to people. Many coffees later, it became apparent that value is realised when we bring UX into DevOps ways of working. Such optimisiation enables flow through a system which accelerates delivery and provides a competitive advantage.
+
+I've put together three reasons why you should optimise UX for Software.
 <br /><br />
 
-## What is UX?
+## Let's start by understanding what UX is...
```

Comment:
You've lost your pithy title - I prefer the original.

`What is UX?` - is easy for me to understand - and decide whether I want to read on.